teams/vso/teams/representatives/engineering/ADRs/using-sign-in-service.md

```diff
@@ -0,0 +1,60 @@
+** Benefits Representative Facing Tools ADR for using Sign-in Service
+
+*** Status: accepted
+
+*** Context
+
+One of the main user stories for our MVP is
+
+> As a VSOfficer: I require a secure login system to verify my accreditation and identity in order to protect Veterans’ Personal Identifiable Information (PII) and Protected Health Information (PHI).
+
+There are two different main options for login systems we can use at the time of writing this, SAML authentication and Sign-in Service. The SAML authentication is
+what is currently used on VA.gov and has a lot of existing infrastructure. Sign-in service is newer and at the time of writing this is only used for the VA mobile app
+and also has some existing infrastructure. Since we are building an MVP at this time we need to choose an authentication system that mostly works off-the-shelf as well
+as beign secure.
+
+It was discussed that one of the main UX and technical features we need for our app is for it to appear completely distinct from VA.gov and that when a VSO logs into
+our app we do NOT want them to be automatically logged into VA.gov. SAML authentication may be able to do this however at the time of writing this it is unclear if
+it CAN actually accomplish this in its current state. Sign-in service also allows us to future-proof our app since it is the reccomendend authentication system
+by the Identity team moving forward.
+
+On the topic of UX the SAML authentication and the sign-in service operate very differently. The SAML Authentication uses a modal that pops up and has various log-in
+methods such as Login.gov and ID.me. The Sign-in service makes use of a Unified Sign In Page that is currently hosted on VA.gov and requres a link to this page, then once
```

Comment:
> various log-in methods such as Login.gov and ID.me

Is this unique to SAML, or will SIS let users choose between Login.gov/ID.me as well?

Reply:
The modal UX is independent from SAML SSOe vs SiS. It is possible to use the modal while also using SiS.